components/courses/app/models/courses/test_task.rb

```diff
@@ -1,7 +1,7 @@
 module Courses
   class TestTask < ApplicationRecord
     self.table_name = 'courses_test_tasks'
-    belongs_to :student, optional: true
+    belongs_to :student
```

Comment:
This isn't interview slot, it's test task. I can't even imagine how there can be test task done without a student.